pulsar-common/src/main/java/org/apache/pulsar/common/policies/data/OffloadPolicies.java

```diff
@@ -39,11 +39,15 @@
     public final static int DEFAULT_OFFLOAD_MAX_THREADS = 2;
     public final static String[] DRIVER_NAMES = {"S3", "aws-s3", "google-cloud-storage", "filesystem"};
     public final static String DEFAULT_OFFLOADER_DIRECTORY = "./offloaders";
+    public final static long DEFAULT_OFFLOAD_OFFLOAD_THRESHOLD = -1;
+    public final static Long DEFAULT_OFFLOAD_OFFLOAD_DELETION_LAG_MS = null;
```

Comment:
Maybe the default value of this config could keep the original value (4 hours).

Reply:
As it's deletion configuration I prefer set the default value to no deletion. WDYT?